sdks/python/apache_beam/utils/windowed_value.py

```diff
@@ -202,10 +202,10 @@ def __ne__(self, other):
     return not self == other
 
   def __hash__(self):
-    return (hash(self.value) +
-            3 * self.timestamp_micros +
-            7 * hash(self.windows) +
-            11 * hash(self.pane_info))
+    return ((hash(self.value) & 0xFFFFFFFFFFFFFFF +
+            3 * (self.timestamp_micros & 0xFFFFFFFFFFFFFF) +
```

Comment:
Could you explain what the bug is here because from what I read Python will truncate the returned result to the first 8 bytes on 64-bit builds (https://docs.python.org/3/reference/datamodel.html#object.__hash__).

Reply:
Yes, but when compiled some C tools will complain about this.